test/turbopack-build-tests-manifest.json

```diff
@@ -5419,10 +5419,10 @@
       "runtimeError": false
     },
     "test/e2e/new-link-behavior/material-ui.test.ts": {
-      "passed": [
+      "passed": [],
+      "failed": [
         "New Link Behavior with material-ui should render MuiLink with <a>"
```

Comment:
Why are these test cases failing now?

Reply:
IIRC, it was timeout, but I think I can revert this